h3i/src/client/connection_summary.rs

```diff
@@ -106,21 +115,105 @@ impl StreamMap {
     /// use quiche::h3::Header;
     /// use std::iter::FromIterator;
     ///
+    /// let mut stream_map = StreamMap::default();
```

Comment:
huh?

Reply:
Ah missed that in the rebase, thanks. I cleaned up another occurrence as well